configuration/concepts/common.csv

```diff
@@ -0,0 +1,74 @@
+9e071d37-ee01-4785-a756-7e4059ffe736,,CIEL:1789,Gestational Age,Age gestationnel,Gestational Age,Age gestationnel,,,ConvSet,N/A,,,Number of Weeks; Number of Days,,,,,,,,
+7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,CIEL:1440,Fetal Heart Rate,Rythme cardiaque fœtal,Fetal Heart Rate,Rythme cardiaque fœtal,,,Misc,Numeric,,,,FALSE,beats/min,120,160,40,250,,
+c4526510-3f10-11e4-adec-0800271c1b75,,CIEL:160091,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
+a0dc5727-c33e-479b-b70e-fb0eb7e45853,,CIEL:146922,Breech,Siège,Breech,Siège,,,Misc,N/A,,,,,,,,,,,
+5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,CIEL:112259,Transverse,Transversale,Transverse,Transversale,,,Misc,N/A,,,,,,,,,,,
+cbafba29-b62a-4675-910a-a6bb161de256,,,Variable,Variable,Variable,Variable,,,Misc,N/A,,,,,,,,,,,
+d47e2e11-355e-4c62-86a7-8ebe45fb4292,,CIEL:160090,Presentation,Présentation,Presentation,Présentation,,,Misc,Coded,,Cephalic;Breech;Transverse;Variable,,,,,,,,,
+7f703275-777c-4237-9d07-36c99ea0966c,,CIEL:153551,Abdominal pain,Douleurs abdominales,Abdominal pain,Douleurs abdominales,,,Misc,N/A,,,,,,,,,,,
+6f8988d1-157a-44cc-ac3b-22b627a57db3,,,Fever,Fièvre,Fever,Fièvre,,,Misc,N/A,,,,,,,,,,,
+e90fa130-09cd-4574-936f-91fca52a9324,,CIEL:125198,Swollen feet,Pieds gonflés,Swollen feet,Pieds gonflés,,,Misc,N/A,,,,,,,,,,,
+8d83ff2b-3f6f-432a-b0cd-5b4f2f1055fd,,CIEL:118939,Vision problems,Troubles de la vision,Vision problems,Troubles de la vision,,,Misc,N/A,,,,,,,,,,,
+4ba98e11-cb3b-4a22-be4e-bb43f3078683,,CIEL:163750,Uterine contractions,Contractions utérines,Uterine contractions,Contractions utérines,,,Misc,N/A,,,,,,,,,,,
+2c1e2efe-74cb-475d-a922-3785e02e7bd3,,CIEL:147232,Vaginal bleeding,Saignements vaginaux,Vaginal bleeding,Saignements vaginaux,,,Misc,N/A,,,,,,,,,,,
+d291cf12-a568-4eec-9f86-9ef55333c3b2,,,Risky Pregnancy,Grossesse à risque,Risky Pregnancy,Grossesse à risque,,,Misc,Coded,,Yes; No,,,,,,,,,
+ee39e8af-12e8-41ae-a1f9-b08acea07587,,,Other Pregnancy Risks,Autres risques de grossesse,Other Pregnancy Risks,Autres risques de grossesse,,,Misc,Text,,,,,,,,,,,
+ca69545b-dfef-4210-b311-dc8a1d309311,,CIEL:540,Anterior,Antérieur,Anterior,Antérieur,,,Misc,N/A,,,,,,,,,,,
+1f82cf3c-ce84-4c98-a35a-c37c45edc5fc,,CIEL:541,Posterior,Postérieur,Posterior,Postérieur,,,Misc,N/A,,,,,,,,,,,
+f5c721f6-7162-4533-9a83-4a4cd6997cb8,,CIEL:542,Lateral,Lateral,Lateral,Lateral,,,Misc,N/A,,,,,,,,,,,
+4d1f668f-f9b7-4e95-8436-53f37e8c643e,,,Other observations,Autres observations,Other observations,Autres observations,,,Misc,Text,,,,,,,,,,,
+b0823b2c-4bc8-2a7c-41b5-7bf3c7f14e82,,,Duration,Durée,Duration,Durée,,,Misc,Numeric,,,,,,,,,,,
+4f6bc54d-6168-881b-86c7-7c4f1d7a4e9e,,CIEL:1074,Months,Mois,Months,Mois,,,Misc,N/A,,,,,,,,,,,
+a4d0c480-923b-49fe-b125-80453675fdea,,CIEL:1734,Years,An(s),Years,An(s),,,Misc,N/A,,,,,,,,,,,
+850c5ae8-38eb-270b-2ff2-e4ade2e95faf,,CIEL:1822,Hours,Heures,Hours,Heures,,,Misc,N/A,,,,,,,,,,,
```

Comment:
in fr:
Heure(s)
Jour(s)
Semaine(s)

in en:
Month(s)
Year(s)
Hour(s)
Day(s)
Week(s)

Reply:
Done